gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDatasetFinder.java

```diff
@@ -170,7 +170,7 @@ protected static boolean getConfigShouldCopyMetadataPath(Properties properties)
   }
 
   /** @return property value or `null` */
-  protected static String getLocationQualifiedProperty(Properties properties, CatalogLocation location, String relativePropName) {
+  public static String getLocationQualifiedProperty(Properties properties, CatalogLocation location, String relativePropName) {
```

Comment:
based on my suggestions to rework the filter predicate ctor params, I'm curious whether this can be accessed entirely within `IcebergPartitionDatasetFinder` and therefore could remain `protected`